.gitignore

```diff
@@ -7,6 +7,8 @@ node_modules
 *.vsix
 **/bin/
 **/obj/
+src/modules/samples/rust/target
+src/modules/samples/rust/Cargo.lock
```

Comment:
Can this be changed to something more generic for anything built using Rust (instead of needing to add each `target/` dir and `Cargo.lock`)?